central/graphql/resolvers/image_cve_core.go

```diff
@@ -40,6 +40,7 @@ func init() {
 				"exceptionCount(requestStatus: [String]): Int!",
 				"images(pagination: Pagination): [Image!]!",
 				"topCVSS: Float!",
+				"cvePublishedDate: Time",
```

Comment:
Just want to make sure this is the same key that appears in the other GraphQL queries, so the UI can re-use code if necessary?

Reply:
key in other query is publishedOn. but it is a different query so not sure if code can be re used

Reply:
I don't mean reusing the query code; I mean the components that take the query result. What is the reason to call the data `cvePublishedDate` here, if the same data is called `publishedOn` in another resolver?

Reply:
ok got it. i can change it to publishedOn